pkg/image/importer/importer.go

```diff
@@ -110,10 +110,18 @@ func (i *ImageStreamImporter) importImages(ctx gocontext.Context, retriever Repo
 		if from.Kind != "DockerImage" {
 			continue
 		}
-		ref, err := api.ParseDockerImageReference(from.Name)
-		if err != nil {
-			isi.Status.Images[i].Status = invalidStatus("", field.Invalid(field.NewPath("from", "name"), from.Name, fmt.Sprintf("invalid name: %v", err)))
-			continue
+		var (
+			err error
+			ref api.DockerImageReference
+		)
+		if from.Name != "*" {
```

Comment:
If you get this you can just skip it.  DockerImage = "*" returns no result.  So up with `from.Kind != "" || from.Name == "*"`